services/indexer/term/index.js

```diff
@@ -62,7 +62,7 @@ class RepoTermLoaderStream extends Transform {
 
     const _loadTerm = (termType, termVal) => {
       // TODO: might not want to be so forceful about lower case
-      termVal = termVal.toLowerCase();
+      if(termVal!=null){termVal = termVal.toLowerCase();}
```

Comment:
No need to squish this into a single line - better to use the same spacing format as the rest of the code, even though this is one simple operation.

More importantly: if `termVal` is `null`, should we be continuing through the rest of this block?